src/entity/player.js

```diff
@@ -16,12 +16,24 @@ class Player extends Entity  {
         this.vx = 0, this.vy = 0;
         this.dirX = 0, this.dirY = 0;
         this.keysPressed = {"left": false, "right": false, "up": false, "down": false, "space": false};
+        this.collidable = true;
+        this.collideLayer = 1;
 
         window.addEventListener('keydown', this.onKeyPress.bind(this));
         window.addEventListener('keyup', this.onKeyUp.bind(this));
 
         // set during load
-        this.board = undefined;
+        this.boardTree = undefined;
+    }
+
+    fireBullet() {
```

Comment:
Looking below, I don't think we're calling this in `Player.attack`

Reply:
Thx!